de.dlr.sc.virsat.model.extension.tests.edit/src-dmf/de/dlr/sc/virsat/model/extension/tests/tests/provider/TestCategoryBeanAItemProvider.java

```diff
@@ -90,7 +90,6 @@ public String getText(Object object) {
 	@Override
 	public void notifyChanged(Notification notification) {
 		updateChildren(notification);
-		super.notifyChanged(notification);

```

Comment:
Why are these statements changed? Why is there a change here in this PR. Looks like a change to the code generator. Would be good to know hat happened. Is it a bad change to the code egenrator, or were the extension.tests not updated with a change on the concept generators. Please clarify, if possible.

Reply:
The eclipse update to 06/2019 lead to an update of the EMF model generator... So it changed all generated DMF files.

Reply:
I reverted all changes in generated files